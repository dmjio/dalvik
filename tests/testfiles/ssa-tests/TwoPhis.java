class TwoPhis {
  static int target;
  static int entry(int x) {
    int i1;
    int i2;
    if(x > 0) {
      i1 = x;
      i2 = 12;
    }
    else {
      i1 = 0;
      i2 = 22;
    }

    target = i1;
    return i2;
  }
}
