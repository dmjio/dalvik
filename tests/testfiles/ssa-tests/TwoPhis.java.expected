SimplePhi { numBlocks = 4
          , phis = [ (0, fromList ["0", "%%arg0"])
                   , (1, fromList ["12", "22"])
                   ]
          , blockPhis = [(0, 2), (1, 2)]
          , assignedPhis = [(0, "target")]
          }